rpm/droidmedia.spec

```diff
@@ -46,7 +46,12 @@ mv droidmedia* droidmedia
 popd
 
 %build
+
+%if "%{?BOARD_QTI_CAMERA_32BIT_ONLY:%{BOARD_QTI_CAMERA_32BIT_ONLY}}%{!?BOARD_QTI_CAMERA_32BIT_ONLY:0}" == "true" 
```

Comment:
%macros are expanded during the first pass of processing .spec, so no Android env is ready at that point